README.md

```diff
@@ -4,11 +4,11 @@
 
 Some guy said:
 
-> How great would it be if we could use well maintained [Postman](https://www.getpostman.com/) collections to generate beautiful documentations with a single command?
+> How great would it be if we could use the well maintained [Postman](https://www.getpostman.com/) collections to generate beautiful documentations with a single command?
```

Comment:
Tu es sûr pour ici ? Il me semblait qu'on évitait d'utiliser le "the" l'expression qui suit est au pluriel.